### PR TITLE
Set log level

### LINKED ISSUE #677: Let turn off logging
Even if you set a logging level to Error or Fatal in gitbase Server.Execute (or even in main) it still prints out INFO+ logs on stdout.

### PR DESCRIPTION
**Pull request by jfontan**
You can disable errors setting the log level to `fatal`.

Fixes #677

### CODE REVIEW

File: cmd/gitbase/command/server.go [hunk 1]
```diff
@@ -107,6 +108,12 @@ func NewDatabaseEngine(
 func (c *Server) Execute(args []string) error {
 	if c.Verbose {
 		logrus.SetLevel(logrus.DebugLevel)
```
Comment (kuba--) [inline cmd/gitbase/command/server.go]: so maybe we don't need verbose anymore. it kind of duplicates log-level.
so by default we'll have info, but if you need more set up to debug.

Comment (jfontan) [reply cmd/gitbase/command/server.go]: It does but I also didn't want to break anything that used `-v`. Maybe we can deprecate it and hide the option from the help but continue supporting it.

Comment (kuba--) [reply cmd/gitbase/command/server.go]: Of course, but right now -v and --log-level are exclusive. In other words, if you specify `-v` then `--log-level` is ignored. Maybe we keep the old `-v` but let --log-level to overwrite it (so move it outside `else`) - WDYT?

Comment (jfontan) [reply cmd/gitbase/command/server.go]: I'll change it to do as you say.

Comment (jfontan) [reply cmd/gitbase/command/server.go]: Changed the behavior and updated documentation.

**Pull request merged**